bears/general/FilenameBear.py

```diff
@@ -38,6 +46,21 @@ def run(self, filename, file,
         """
         head, tail = os.path.split(filename)
         filename_without_extension, extension = os.path.splitext(tail)
+
+        if file_naming_convention is None:
+            self.warn('Please specify a file naming convention explicitly. '
+                      'Using the default "snake" naming convention...')
+            file_naming_convention = 'snake'
```

Comment:
Oh and other point: I would add to the warning message:
```
Please specify a file naming convention explicitly or use "auto".
```
"explicitly specifying" would mean for me that you shouldn't use "auto", but instead `snake`, `camel` or others